src/copy.jl

```diff
@@ -137,3 +137,10 @@ function Base.copy(model::AbstractModel)
     new_model, _ = copy_model(model)
     return new_model
 end
+
+# Calling `deepcopy` over a JuMP model is not supported, nor planned to be
+# supported, because it would involve making a deep copy of the underlying
+# solver (behind a C pointer).
+function Base.deepcopy(::Model)
+    error("JuMP.AbstractModel does not support deepcopy as the reference to the underlying solver cannot be deep copied, use copy instead.")
```

Comment:
Use backsticks around code, i.e. `JuMP.AbstractModel`, `deepcopy`, `copy`

Reply:
In the error message string you say? Or in the comment? Or both?
I did not know the error message made use of this formatting.

Reply:
> I did not know the error message made use of this formatting.

It does not but we expect the developer reading the error message to understand the meaning. That is consistent with what we do in other places in JuMP and MOI.